src/plugins/intel_cpu/src/nodes/scaled_attn.cpp

```diff
@@ -1034,7 +1034,7 @@ struct ScaledDotProductAttention::AttentionExecutor : public ScaledDotProductAtt
             B = k_input.size(0);
             L1 = k_input.size(1);
             auto Hk = present_key.size(1);
-            S = present_value.size(3);
+            S = present_value.size(3) - (present_value.m_dt == ov::element::Type_t::u8 ? 8 : 0);
```

Comment:
Same. Not obvious change. Why we need it?

Reply:
Added the following comment:
```
// The layout for per token per head for u8 kv cache:
// |scale(f32)|zeropoint(f32)|quantized feature(u8,idx_1)|quantized feature(u8,idx_2)|...|quantized feature(u8,idx_S)|
// The actual size needs to deduct scale and zeropoint.
```